modules/about-etcd-encryption.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * authentication/encrypting-etcd.adoc
+
+[id="about-etcd_{context}"]
+= About etcd encryption
```

Comment:
On a recent minimalism call, Ingrid suggested removing "about" from modules titles because it doesn't add much value.

Reply:
I did hear that, but I don't necessarily agree with it. I think it helps to make it clear that this is overview/conceptual information. Just putting "etcd encryption" would look odd to me. Unless we make a team decision to remove these instances, I'd like to keep this as is now.

Reply:
I'll add it to the team meeting agenda.